cylc/flow/scripts/remove.py

```diff
@@ -18,7 +18,26 @@
 
 """cylc remove [OPTIONS] ARGS
 
-Remove one or more task instances from a running workflow.
+Remove task instances from a running workflow and the workflow's history.
+
+This removes the task(s) from any specified flows. The task will still exist,
+just not in the specified flows, so will not influence the evolution of
+the workflow in those flows.
```

Comment:
```suggestion
This removes the task(s) from specified flows. The task will still exist,
but it will not influence the evolution of the workflow in those flows.
```

Reply:
Now outdated.